lib/graphql/client/apollo-client.ts

```diff
@@ -1,12 +1,14 @@
+'use client'
+
 import { ApolloClient, InMemoryCache, from, createHttpLink } from "@apollo/client";
-import { authLink, retryLink } from "../graphqlHelper";
+import { errorLink, authLink, retryLink } from "@/lib/graphql/graphqlHelper";
```

Comment:
Switched the `apollo-client` to use the shared `errorLink`, `authLink`, and `retryLink` with the `apollo-wrapper`. This file is still needed if we want to use a different way to query graphQL.